internal/docker/deployer.go

```diff
@@ -283,7 +275,7 @@ func deployImage(
 		Mounts:     mounts,
 	}, &network.NetworkingConfig{
 		EndpointsConfig: map[string]*network.EndpointSettings{
-			contextStr: {
+			"complement_" + pkgNamespace + "_" + blueprintName: {
```

Comment:
Why aren't we using the `contextStr` anymore?

Reply:
This needs to be the name of the network, which is not the case here.

We could also use the contextStr as the name of the network in `createNetworkIfNotExists`, or makes `createNetworkIfNotExists` returns both the ID and the network name and store both in `Deployer`.

The latter looks like the cleaner way to do it so leaning towards that.

Reply:
Actually it seems like we can just use the name, ID is optional, at least with Podman.
I've push a change to return and use the network name instead of the ID, let's see if Docker is happy.